submitted_models/ctu_cras_norlab_absolem_sensor_config_1/urdf/boson_thermocam.xacro

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<root xmlns:xacro="http://www.ros.org/wiki/xacro">
+    <xacro:property name="thermocam_mass" value="0.024" scope="global" /> <!-- https://flir.netx.net/file/asset/15754/original/attachment -->
+
+    <xacro:macro name="boson_thermocam" params="name simulate:=1 visualize:=0">
+        <!-- Thermal camera - FLIR Boson 320 20320H092 -->
+        <link name="$(arg prefix)${name}">
+            <visual>
+                <origin xyz="0 0 ${0.021/2}" />
+                <geometry>
+                    <mesh filename="package://ctu_cras_norlab_absolem_sensor_config_1/meshes/flir_boson.dae" />
+                </geometry>
+            </visual>
+            <collision>
+                <origin xyz="${0.043/2 - 0.006} 0 0" />
+                <geometry>
+                    <box size="0.043 0.021 0.021"/>
+                </geometry>
+            </collision>
+            <xacro:box_inertial mass="${thermocam_mass}" width="0.021" height="0.021" depth="0.043" xyz="${0.043/2 - 0.006} 0 0" />
+        </link>
+        <xacro:if value="${simulate}">
+            <gazebo reference="$(arg prefix)${name}">
```

Comment:
Please adjust the thermal camera parameters to be consistent with the [updated values](https://github.com/osrf/subt/blob/ba0f2ec00e1db43197da0faf824cfda0ff8e6813/submitted_models/ctu_cras_norlab_marv_sensor_config_1/urdf/boson_thermocam.xacro) in #910.